bors.toml

```diff
@@ -0,0 +1,4 @@
+status = [ "cargo-registry-cache", "cargo-format", "cargo-sort", "cargo-check", "cargo-clippy","cargo-test", "cargo-tarpaulin","test-android-libs","build-ios-libs", "test-wasm-libs","build-linux-lib","flutter-format","flutter-checks","flutter-build-example"]
+timeout_sec = 36000 # ten hours
```

Comment:
This is so high because the timeouts in the CI are working and is to avoid to update this evry time we need to update a timeout in the ci?

Reply:
Exactly, if any of the previous status fail then it will fail. So it isn't so critical. It is just if in case it gets stuck trying to merge, then it will fail. But I can do it less? But how less? 5 hours?

Reply:
ok, let's do 5